README.md

```diff
@@ -19,20 +19,25 @@ Arctic has been under active development at [Man Group](https://www.man.com/) si
 
 ---
 
-## :mega: ArcticDB coming soon! :mega:
+## :mega: ArcticDB! :mega:
 
-[ArcticDB](https://www.man.com/man-group-brings-powerful-dataframe-database-product-arcticdb-to-market-with-bloomberg), Man Group's high-performance Python-native DataFrame database will soon be available on GitHub. ArcticDB is the next generation of Arctic and is designed to be a drop-in replacement for Arctic, providing a platform that is:
+[ArcticDB](https://www.man.com/man-group-brings-powerful-dataframe-database-product-arcticdb-to-market-with-bloomberg), Man Group's high-performance Python-native
```

Comment:
First link should point to http://arcticdb.io rather than the press release IMO. As it's the first thing people will click...